wg-meetings/2018-04-18.md

```diff
@@ -0,0 +1,74 @@
+# Node.js Foundation Diagnostics Working Group Meeting 2018-03-21 Minutes
+
+## Attendees
+ - @mike-kaufman
+ - @mrkmarron
+ - @tedsuo
+ - @ofrobots
+ - @mcollina
+ - @hekike
+
+
+## Links
+- **This Meeting**:
+  - Minutes:  https://github.com/nodejs/diagnostics/blob/master/wg-meetings/2018-04-18.md
+  - Notes: https://docs.google.com/document/d/14dRC5OT6gD0jP8YG6a7Iunahw86CaGKj2DmQsggq0Jg/edit
+  - You Tube: http://youtu.be/hePMyxKiCf0
+- **Previous Meeting**: 
+  - Minutes:  https://github.com/nodejs/diagnostics/blob/master/wg-meetings/2018-04-04.md
+  - Notes: https://docs.google.com/document/d/1gRRyLfakH69lnpZMMcaBsB5c8WOO5LG7utNm7mrLEoo/edit
+  - You Tube: http://youtu.be/Yh4pdMmM330
+
+## Agenda
+
+Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/diagnostics
+
+ - Diagnostics Channel - Next steps [#180](https://github.com/nodejs/diagnostics/issues/180)
+   - @qard added link to some prototype code for pub/sub channel
+   - no other progress
+ - Visualizing Async Context [#179](https://github.com/nodejs/diagnostics/issues/179)
+  
+ - Diag WG Deep Dives - topics [#168](https://github.com/nodejs/diagnostics/issues/168)
+  - please update if anyone has any topics 
+
+ - Node CPU Profiling Roadmap [#148](https://github.com/nodejs/diagnostics/issues/148)
+   - PR in v8 was merged last week
+   - will be available in v8 6.7 release
+   - will node v10 ship with 6.6 or 6.7?  Unknown

```

Comment:
v10.0.0 will ship with 6.6 and it's going to be updated to 6.7 eventually (https://github.com/nodejs/node/issues/19354#issuecomment-373157341).